specifications/api/swagger.yaml

```diff
@@ -2940,3 +2997,29 @@ paths:
       parameters:
         - *parametersAddressId
       responses: *responsesInspectAddress
+
+  /settings:
+    put:
+      operationId: putSettings
+      tags: ["Settings"]
+      summary: Settings
```

Comment:
How about "Update" or "Set"? Otherwise it looks a bit strange:

![Screenshot from 2020-10-06 15-26-37](https://user-images.githubusercontent.com/42900201/95207629-5a140280-07e8-11eb-888f-491a6e294a54.png)